src/react-elastic-carousel/components/styled/Button.js

```diff
@@ -1,13 +1,15 @@
 import styled from "styled-components";
 
 export default styled.button.attrs(({ type = "button" }) => ({ type }))`
+  box-sizing: border-box;
```

Comment:
I believe that this line is not needed to fix issue on iOS. Setting padding to 0 is enough 👍

Reply:
yeah, most browser defaults to that but AFAIK some UI lib has a general rule that might override it.